editor.planx.uk/src/pages/FlowEditor/components/PreviewBrowser.tsx

```diff
@@ -196,19 +204,39 @@ function AlteredNodesSummaryContent(props: any) {
       {changeSummary["portals"].length > 0 && (
         <Box pt={2}>
           <Typography variant="body2">{`This includes recently published changes in the following nested services:`}</Typography>
-          <ul>
+          <List sx={{ listStyleType: "disc", marginLeft: 4 }}>
             {changeSummary["portals"].map((portal, i) => (
-              <li key={i}>
-                {useStore.getState().canUserEditTeam(portal.split("/")[0]) ? (
-                  <Link href={`../${portal}`} target="_blank">
-                    <Typography variant="body2">{portal}</Typography>
-                  </Link>
-                ) : (
-                  <Typography variant="body2">{portal}</Typography>
-                )}
-              </li>
+              <ListItem key={i} disablePadding sx={{ display: "list-item" }}>
+                <ListItemText
+                  primary={
+                    useStore
+                      .getState()
+                      .canUserEditTeam(portal.text.split("/")[0]) ? (
+                      <Link href={`../${portal.text}`} target="_blank">
+                        <Typography variant="body2">{portal.text}</Typography>
+                      </Link>
+                    ) : (
+                      <Typography variant="body2">{portal.text}</Typography>
+                    )
+                  }
+                  secondary={
+                    <>
+                      <Typography variant="body2" fontSize="small">
+                        {`Last published ${formatDistanceToNow(
+                          new Date(portal.publishedAt),
+                        )} ago by ${portal.publishedBy}`}
+                      </Typography>
+                      {portal.summary && (
+                        <Typography variant="body2" fontSize="small">
+                          {portal.summary}
+                        </Typography>
+                      )}
+                    </>
+                  }
+                />
+              </ListItem>
             ))}
-          </ul>
+          </List>
```

Comment:
This looks like this :
![Screenshot from 2024-03-01 16-49-00](https://github.com/theopensystemslab/planx-new/assets/5132349/97e2f600-b652-446d-8862-6fcf699d3d69)

Reply:
This is such a nice outcome 🎉